SimPEG/MT/Utils/dataUtils.py

```diff
@@ -44,19 +44,19 @@ def rotateData(MTdata,rotAngle):
     return MT.Data.fromRecArray(outRec)
 
 
-def appResPhs(freq,z):
+def appResPhs(freq, z):
     app_res = ((1./(8e-7*np.pi**2))/freq)*np.abs(z)**2
```

Comment:
I used the values for mu_0 and omega and simplified. I need to clean out the Utils and fix things like this. 

Should use the constants here, would be much clearer.